src/components/radios/template.njk

```diff
@@ -18,6 +18,11 @@
   {% endif %}
 {% endfor %}
 
+{% set userDefinedClasses = ' ' + params.classes if (params.classes) %}
```

Comment:
Given we [already know whether there's a conditional or not](https://github.com/alphagov/govuk-frontend/pull/970/files#diff-5eee27b7e0c518d51503bac6513ad0edR17), what do we think about adding another class (e.g. `govuk-radios--conditional`) to the wrapper if `isConditional` is true?

We don't need to know if they're using the inline modifier or not, because that class wouldn't do anything by itself, which would allow us to simplify this a lot.

We could then just do something like…

```scss
  // Inline variant
  .govuk-radios--inline {
    @include mq ($from: tablet) {
      @include govuk-clearfix;

      .govuk-radios__item {
        margin-right: govuk-spacing(4);
        float: left;
        clear: none;
      }
    }
  }

  // Prevent inline modifier being used with conditional reveals
  .govuk-radios--conditional.govuk-radios--inline {
    .govuk-radios__item {
      float: none;
    }
  }
```

(Potentially we could just use `:not` if we make the inline modifier IE9+ only, and fall back in IE8?)

Thoughts?

Reply:
Alternatively, if we want a more rigorous approach that would work for users who aren't using the macros, we could style based on `data-module="radios"` 

```
  [data-module="radios"].govuk-radios--inline {
    .govuk-radios__item {
      float: none;
    }
  }
```

It means we're styling based on a data attribute, which is a bit odd… but might be worth considering in this case.

Reply:
This seems like a better approach since it won't force us to document something we don't want people to do.

Reply:
FWIW the data module attribute is not a requirement for our JavaScript since you can initialise based on any selector, so by doing that 2nd approach we'd force people to use it.

So I'm in favour of your first proposal

Reply:
ok, let me implement the first proposal

Reply:
implemented proposed solution